packages/transformers/react-refresh-wrap/src/ReactRefreshWrapTransformer.js

```diff
@@ -3,6 +3,11 @@
 import path from 'path';
 import {Transformer} from '@parcel/plugin';
 
+const FILENAME = // $FlowFixMe[prop-missing]
+  process.env.PARCEL_BUILD_REPL && process.browser
+    ? '/app/__virtual__/@parcel/transformer-react-refresh-wrap/src/ReactRefreshWrapTransformer.js'
+    : __filename;
```

Comment:
Wonder if we should have a lint rule or something to ensure we don't accidentally add `__filename` or `__dirname` again? Also, I guess the default browser replacement that parcel does for those doesn't work in this case?

Reply:
I changed this now to simply check for these `/app/packages/runtimes/js/src/` paths in the resolver.